internal/stack/_static/kibana.yml.tmpl

```diff
@@ -137,3 +133,185 @@ xpack.fleet.outputs:
         {{ indent $agent_key "        " }}
   {{ end }}
 {{ end }}
+
+{{ $serverless_enabled := fact "serverless_enabled" }}
+{{ $serverless_project_type := fact "serverless_project_type" }}
+{{ if eq $serverless_enabled "true" }}
+  {{ if eq $serverless_project_type "observability" }}
+# Observability Project config
```

Comment:
AFAIK, just setting `serverless: oblt` should be enough to load all other settings (if you use the Kibana Serverless docker image).